x/twap/logic.go

```diff
@@ -182,7 +182,7 @@ func recordWithUpdatedAccumulators(record types.TwapRecord, newTime time.Time) t
 		return record
 	}
 	newRecord := record
-	timeDelta := newTime.Sub(record.Time)
+	timeDelta := types.CanonicalTimeMs(newTime) - types.CanonicalTimeMs(record.Time)
```

Comment:
can we add a one line comment above here on why we use canonical time for the time delta?

Reply:
Since this is called in multiple places, I elaborated in the method spec instead:
[937d148](https://github.com/osmosis-labs/osmosis/pull/4359/commits/937d148fcf078eb3e81dbd35a606104ff9eb88c4)